modules/configuring-egress-proxy-edns-operator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-configuring-cluster-wide-egress-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-cluster-wide-proxy_{context}"]
+= Configuring the cluster-wide proxy on the External DNS Operator
```

Comment:
```suggestion
= Configuring the cluster-wide egress proxy on the External DNS Operator
```

Reply:
As discussed - https://github.com/openshift/openshift-docs/pull/57402#discussion_r1148746649, opting for using the cluster-wide proxy instead of cluster-wide egress proxy